packages/sign_in_with_apple/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -374,7 +378,7 @@
 					"$(inherited)",
 					"$(PROJECT_DIR)/Flutter",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.example.signInWithAppleExample;
+				PRODUCT_BUNDLE_IDENTIFIER = de.lunaone.flutter.signinwithappleexample;
```

Comment:
Should we maybe create an ay specific bundle identifier that we can just use with our AY addresses?

Reply:
Especially with the automatic signing, I didn't want to auto-update anything in our production account right from Xcode, so I used another developer account for testing.

If you think we can create a test app and share the certificates manually, that would be great. We will definitely have to start this a bunch of times in the future for debugging.

Reply:
Follow up @ https://github.com/aboutyou/dart_packages/issues/66